sites/partners/src/components/listings/ListingFormActions.tsx

```diff
@@ -333,7 +357,26 @@ const ListingFormActions = ({
       </Grid.Cell>
     )
 
-    const lotteryResultsButton = (elements) => {
+    const elements = []
+
+    //lottery buttons for listing edit view
+    const updateLotteryResultsButton = () => {
+      const lotteryResults = listing?.listingEvents?.find(
+        (event) => event.type === ListingEventsTypeEnum.lotteryResults
+      )
+      if (lotteryResults) {
+        elements.push(editPostedResultsButton(lotteryResults))
+      } else if (
+        listing.status === ListingsStatusEnum.closed &&
+        // dwy-specific lottery logic to only show post results if showLottery flag is enabled
+        !listing?.lotteryOptIn &&
+        process.env.showLottery
```

Comment:
This logic was verified with product since Doorway uses the env to hide anything related to lottery but core uses it to more specifically for the lottery tab. For example, show lottery is false in hba but they can still post lottery results manually as was existing behavior